<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneEditorWidget</class>
 <widget class="QWidget" name="SceneEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scene Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>294</width>
        <height>719</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="graphics_listview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QComboBox" name="add_graphics_combobox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Add...</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>---</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>point</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>node points</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>data points</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>element points</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>lines</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>surfaces</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>contours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>streamlines</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_graphics_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="GraphicsEditorWidget" name="graphics_editor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>frame</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsEditorWidget</class>
   <extends>QWidget</extends>
   <header>opencmiss/neon/ui/zincwidgets/graphicseditorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>graphics_listview</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SceneEditorWidget</receiver>
   <slot>graphicsListItemClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_graphics_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>SceneEditorWidget</receiver>
   <slot>addGraphicsEntered(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_graphics_button</sender>
   <signal>clicked()</signal>
   <receiver>SceneEditorWidget</receiver>
   <slot>deleteGraphicsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>graphicsListItemClicked(QModelIndex)</slot>
  <slot>addGraphicsEntered(QString)</slot>
  <slot>deleteGraphicsClicked()</slot>
 </slots>
</ui>
